bazel/deps.bzl

```diff
@@ -251,8 +251,8 @@ def stratum_deps():
     if "com_github_opennetworkinglab_sdklt" not in native.existing_rules():
         http_archive(
             name = "com_github_opennetworkinglab_sdklt",
-            sha256 = "38a59fe2db5122dd76fcbed234c68c59ccfdb68890199b4b891aeb86817713f4",
-            urls = ["https://github.com/opennetworkinglab/SDKLT/releases/download/r69/sdklt-4.14.49.tgz"],
+            sha256 = "f86919458e20f417c8de7373963bc077b7063e9820539e7f5997cc4f6fe04ce0",
+            urls = ["https://github.com/opennetworkinglab/SDKLT/releases/download/r140/sdklt-4.14.0.tgz"],
```

Comment:
I think there is a typo of the release filename, should be `sdklt-4.19.0.tgz`, right?

Reply:
good, catch. Right here: 
https://github.com/opennetworkinglab/SDKLT/blob/9aec81546653a232ce733bde32bacec55e0dc421/.circleci/build-sdklt.sh#L29

Reply:
Now this kernel check makes a lot more sense to me
`#if LINUX_VERSION_CODE < KERNEL_VERSION(4,14,0)`

I was wondering why we didn't need that for ONL 4.14.